### ISSUE TITLE
Training Fails using SAEHD [it doesn`t work]

### ISSUE THREAD

**Issue by Arsenii101** [iperov/DeepFaceLab#5517]
During 6th step "train SAEHD" the programm didn`t load any packed faces and just stopped. After reboot it loaded packed faces, but still nothing happens. Can anyone help please with that problem?

Running trainer.

[new] No saved models found. Enter a name of a new model : example
example

Model first run.

Choose one or several GPU idxs (separated by comma).

[CPU] : CPU
  [0] : GeForce GTX 1650 Ti

[0] Which GPU indexes to choose? : 0
0

[0] Autobackup every N hour ( 0..24 ?:help ) : 0
0
[n] Write preview history ( y/n ?:help ) : n
[300000] Target iteration : 3000000
3000000
[y] Flip SRC faces randomly ( y/n ?:help ) : y
[y] Flip DST faces randomly ( y/n ?:help ) : y
[5] Batch_size ( ?:help ) : 8
8
[256] Resolution ( 64-640 ?:help ) : 256
256
[f] Face type ( h/mf/f/wf/head ?:help ) : f
f
[df] AE architecture ( ?:help ) : df
df
[256] AutoEncoder dimensions ( 32-1024 ?:help ) : 256
256
[64] Encoder dimensions ( 16-256 ?:help ) : 64
64
[64] Decoder dimensions ( 16-256 ?:help ) : 64
64
[22] Decoder mask dimensions ( 16-256 ?:help ) : 22
22
[n] Eyes and mouth priority ( y/n ?:help ) : n
[y] Uniform yaw distribution of samples ( y/n ?:help ) : y
[n] Blur out mask ( y/n ?:help ) : n
[y] Place models and optimizer on GPU ( y/n ?:help ) : y
[y] Use AdaBelief optimizer? ( y/n ?:help ) : y
[n] Use learning rate dropout ( n/y/cpu ?:help ) : n
n
[y] Enable random warp of samples ( y/n ?:help ) : y
[0.0] Random hue/saturation/light intensity ( 0.0 .. 0.3 ?:help ) : 0.0
0.0
[0.0] GAN power ( 0.0 .. 5.0 ?:help ) : 0.0
0.0
[0.0] 'True face' power. ( 0.0000 .. 1.0 ?:help ) : 0.0
0.0
[0.0] Face style power ( 0.0..100.0 ?:help ) : 0.0
0.0
[0.0] Background style power ( 0.0..100.0 ?:help ) : 0.0
0.0
[none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) : none
none
[y] Enable gradient clipping ( y/n ?:help ) : y
[y] Enable pretraining mode ( y/n ?:help ) : y
Initializing models: 100%|###############################################################| 5/5 [00:03<00:00,  1.40it/s]
Loaded 15843 packed faces from C:\Users\yurch\Downloads\DeepFaceLab_NVIDIA_up_to_RTX2080Ti\_internal\pretrain_faces
Sort by yaw: 100%|##################################################################| 128/128 [00:00<00:00, 269.41it/s]
Sort by yaw: 100%|##################################################################| 128/128 [00:00<00:00, 252.26it/s]

**Comment by zaiyicheng** [comment]
try change Batch_size to 4

**Comment by mrlunk** [comment]
> try change Batch_size to 4

THIS worked for me !!!
Thanks for the tip !

Peter Lunk.

**Comment by ghost** [comment]
Issue solved / already answered (or it seems like user error), please close it.
